prosessering-core/src/main/kotlin/no/nav/familie/prosessering/internal/TaskStepExecutorService.kt

```diff
@@ -32,14 +31,10 @@ class TaskStepExecutorService(@Value("\${prosessering.maxAntall:10}") private va
         log.debug("Poller etter nye tasks")
         val pollingSize = calculatePollingSize(maxAntall)
 
-        if (pollingSize > minCapacity) {
-            val tasks = taskService.finnAlleTasksKlareForProsessering(PageRequest.of(0, pollingSize))
-            log.trace("Pollet {} tasks med max {}", tasks.size, maxAntall)
+        val tasks = taskService.finnAlleTasksKlareForProsessering(PageRequest.of(0, pollingSize))
```

Comment:
Denne kommer feile hvis pollingSize er 0

Reply:
Takk for info